app/views/shf_applications/edit.html.haml

```diff
@@ -19,6 +19,10 @@
                            company_numbers:  @company_numbers }
 
         = render 'application/required_fields'
-        = f.submit t('.submit_button_label')
+
+        = f.submit t('.submit_button_label'), disabled: true, class: 'app-submit'
+
+        .submit-button-explain.strong
```

Comment:
But `strong` _is_ a CSS selector :)

Anyway, I added this because our current styling results in very faint text (very little contrast between the text and they background colors), and making it bold alleviates that _a little_, but not enough.

I think of this as temporary since @thesuss will adjust with her redesign in any case.